src/searchee.ts

```diff
@@ -160,3 +172,106 @@ export async function createSearcheeFromPath(
 		length: totalLength,
 	});
 }
+
+export function getKeyMetaInfo(stem: string, isAnime: boolean): string {
+	const resM = stem.match(RES_STRICT_REGEX)?.groups?.res;
+	const res = resM ? `.${resM}` : "";
+	const sourceM = sourceRegexParse(stem);
```

Comment:
it looks like these are only going to find WEB-DLs - what about BluRays or other sources like HDTV?

Reply:
The main reason `SOURCE_REGEX` was added is for ensemble aggregating episodes. Individual episodes are unlikely to be bluray, that can could only be a negative. I think HDTV|SDTV could be added but those are rare and can be missing from names so I ignored them like we do for resolutions below 720p.

`SOURCE_REGEX` is being used to reject in `decide.js`, but that's just because it's available. I rather the extra snatches than risk false negatives. It will be caught by fuzzy size and release group anyways since these highly affected by the source.